api/client/client.go

```diff
@@ -0,0 +1,69 @@
+package client
```

Comment:
You have multiple layers of abstraction here, is this really needed? Most of the methods are just wrappers around the GRPC client.

Reply:
At first I used directly the stub to call the RPC, but then started to implement the streaming and I feel like it needs a proper wrapper to hide the implementation from main